Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/BehaviorData.xml

```diff
@@ -7174,4 +7174,73 @@
+        <Name value="Button/Name/AP_ImpalingStrikeStun"/>
+        <Tooltip value="Button/Tooltip/AP_ImpalingStrikeStun"/>
+        <Alignment value="Negative"/>
+        <BehaviorCategories index="Stun" value="1"/>
+        <EditorCategories value="AbilityorEffectType:Units"/>
+        <Duration value="2"/>
+        <Modification>
+            <StateFlags index="Stun" value="1"/>
+        </Modification>
+    </CBehaviorBuff>
+    <CBehaviorBuff id="AP_TyrannozorTyrantsProtectionCaster">
+        <InfoFlags index="Hidden" value="1"/>
+        <InfoIcon value="Assets\Textures\btn-ability-zerg-rapidregeneration.dds"/>
+        <EditorCategories value="Race:Zerg,AbilityorEffectType:Units"/>
+        <DisableValidatorArray value="AP_HaveTyrannozorTyrantsProtection"/>
+        <Period value="0.5"/>
+        <PeriodicEffect value="AP_TyrannozorTyrantsProtectionSearch"/>
+    </CBehaviorBuff>
+    <CBehaviorBuff id="AP_TyrannozorTyrantsProtectionTarget">
+        <InfoIcon value="Assets\Textures\btn-ability-zerg-dehaka-tyrannozor-tyrantprotection.dds"/>
+        <Name value="Button/Name/AP_TyrannozorTyrantsProtectionTarget"/>
+        <Tooltip value="Button/Tooltip/AP_TyrannozorTyrantsProtectionTarget"/>
+        <EditorCategories value="Race:Zerg,AbilityorEffectType:Units"/>
+        <Duration value="0.6"/>
+        <Modification>
+            <LifeArmorBonus value="2"/>
+        </Modification>
+    </CBehaviorBuff>
+    <CBehaviorBuff id="AP_ChitinousPlatingIndicator">
+        <!-- Indicator of passive upgrade - full command card -->
+        <!--        <Alignment value="Positive"/>-->
+        <BehaviorFlags index="Permanent" value="1"/>
+        <DisableValidatorArray value="AP_HaveChitinousPlating"/>
+        <InfoIcon value="Assets\Textures\btn-upgrade-zerg-chitinousplating.dds"/>
+        <Name value="Button/Name/AP_ChitinousPlating"/>
+        <Tooltip value="Button/Tooltip/AP_ChitinousPlating"/>
+    </CBehaviorBuff>
+    <CBehaviorBuff id="AP_AnabolicSynthesisIndicator">
+        <!-- Indicator of passive upgrade - full command card -->
+        <!--        <Alignment value="Positive"/>-->
+        <BehaviorFlags index="Permanent" value="1"/>
+        <DisableValidatorArray value="AP_HaveAnabolicSynthesis"/>
+        <InfoIcon value="Assets\Textures\btn-upgrade-zerg-anabolicsynthesis.dds"/>
+        <Name value="Button/Name/AP_AnabolicSynthesis"/>
+        <Tooltip value="Button/Tooltip/AP_AnabolicSynthesis"/>
+    </CBehaviorBuff>
+    <CBehaviorBuff id="AP_MonarchBladesIndicator">
+        <!-- Indicator of passive upgrade - full command card -->
+        <!--        <Alignment value="Positive"/>-->
+        <BehaviorFlags index="Permanent" value="1"/>
+        <DisableValidatorArray value="AP_HaveHotSMonarchBlades"/>
+        <InfoIcon value="Assets\Textures\btn-upgrade-zerg-monarchblades.dds"/>
+        <Name value="Button/Name/AP_MonarchBlades"/>
+        <Tooltip value="Button/Tooltip/AP_MonarchBlades"/>
+    </CBehaviorBuff>

```

Comment:
These two behaviours are assigned but never used; they can be removed.

It seems the intent was to use these to signal that a passive icon should appear, but that's not necessary (only the upgrade and requirement/node looking at it are).

These behaviours are also the only place the corresponding validators are used, so those can also be removed.

Reply:
This was the result of moving two passives to behaivors, I think this is necessary for that?

Reply:
Ah, it's the indicator! My bad, this is likely still necessary, then.